stylize/clang_formatter.py

```diff
@@ -34,3 +35,14 @@ def run(self, args, filepath, check=False):
             proc.communicate()
             md5_after = file_md5(filepath)
             return md5_before != md5_after
+
+    def get_command(self):
+        if os.system("command -v clang-format >/dev/null") == 0:
```

Comment:
Oh thats definitely better to use in this case!
I don't think there's a `compgen` python builtin, its pretty obscure, and I'm only able to use it because it's a bash builtin... For some reason it's not even in zsh :confounded: :